rtos/rtx/TARGET_CORTEX_M/cmsis_os.h

```diff
@@ -82,7 +82,12 @@
 
 #endif
 
+#if defined (MBED_CFG_DEBUG_OPTIONS_COVERAGE)
+/* GCOV works with stack size approx. 32K. */
+#define DEFAULT_STACK_SIZE         (WORDS_STACK_SIZE*4*8)
```

Comment:
It might be worth checking stack usage with a debugger to see how big this stack needs to be.